akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -348,5 +357,14 @@ abstract class JournalSpec(config: Config)
 
       }
     }
+
+    optional(flag = supportsReplayOnlyLast) {
+      "replay only last when fromSequenceNr is -1" in {
+        journal ! ReplayMessages(-1, Long.MaxValue, Long.MaxValue, pid, receiverProbe.ref)
```

Comment:
fromSequenceNr = -1 will be a special magic that a plugin can look for and only replay last event.
that will still work with old plugins, in the way that they will replay all events